TWLight/applications/forms.py

```diff
@@ -200,6 +201,10 @@ def _add_user_data_subform(self, user_data):
                 user_data_layout.append(datum)
 
             self.helper.layout.append(user_data_layout)
+            self.helper.layout.append(HTML('<p><small><i>Your personal data '
```

Comment:
here's a reasonable way to get this bit of the interface captured for translation without reworking the way we're building the forms:
```
            # Translators: This this note appears in a section of a form where we ask users to enter info (like country of residence) when applying for resource access.
            self.helper.layout.append(HTML(_('<p><small><i>Your personal data '
                'will be processed according to our <a href="{terms_url}">'
                'privacy policy</a>.</i></small></p>'.format(
                    terms_url=reverse('terms')))))
```